content/en/blog/2023/notation-completes-security-audit.md

```diff
@@ -0,0 +1,56 @@
+* Insufficient Verification of Fetched Artifact Descriptor, ADA-NOT-23-6, aka [CVE-2023-33959](https://github.com/notaryproject/notation-go/security/advisories/GHSA-xhg5-42rf-296r), fixed in Notation-v1.0.0-RC.6.
+* Denial of Service from Resource Exhaustion in `notation inspect`, ADA-NOT-23-7, fixed in Notation-v1.0.0-RC.6.
+ 
+### Details by category of Findings
+ 
+- Endless data attack can cause resource exhaustion leading to Denial of Service attack (ADA-NOT-23-1, ADA-NOT-23-2, and ADA-NOT-23-7)
+
+This issue was initially reported for the `notation list` command which lists all signature artifacts associated with a signed image (OCI artifact), but this can also affect other CLI commands such as `notation inspect`, or `notation verify` which pulls all signatures associated with an image. Refer to [CVE-2023-33957](https://github.com/notaryproject/notation/security/advisories/GHSA-9m3v-v4r5-ppx7) and [CVE-2023-33958](https://github.com/notaryproject/notation/security/advisories/GHSA-rvrx-rrwh-r9p6) for details. The concern was a threat actor could cause denial of service attack by associating large number of signatures to an OCI artifact, such as a container image, and causing Notation to endlessly enumerate all signatures. The fix was relatively straightforward to have a default configurable maximum limit of 100 signatures that Notation CLI will enumerate for any given operation. Users can adjust this number for their unique needs.
+
+- Overwriting a Global Variable (ADA-NOT-23-3)
+
+Notation overwrites a global import identifier in the verification command. There is no current way to exploit this issue, but it could lead to undefined behavior of Notation in the future, if a contributor adds code that allows an attacker to trigger an issue. The issue is flagged informational since we have found no attack vector.
+ 
+- Insufficient documentation and CLI commands (ADA-NOT-23-4 and ADA-NOT-23-5)
+
+Notation maintainers have improved documentation to include security best practices for deploying Notation, developing and managing plugins, validating plugins that are downloaded from a trusted source, and renamed `plain-http` flag to `insecure-registry` to clarify its implications that authenticating to registries over HTTP is insecure and should only be used for testing purposes. 
+ 
+- Validating unintended artifact (ADA-NOT-23-6)
+
+This issue allowed threat actors, who have compromised the registry, to sign or verify the artifact other than intended one. Refer to [CVE-2023-33959](https://github.com/notaryproject/notation-go/security/advisories/GHSA-xhg5-42rf-296r) for more details. The has been fixed in the notation-go library to validate that the descriptor signed or verified by Notation is the one provided by user.
+
+## Fuzzing
```

Comment:
We had another blog post about the fuzzing. What is the new information we are communicating here?

Reply:
@toddysm  I think the fuzzing audit is a part of the security audit. Fuzzing is also included in other project security audit blog post like Istio: https://istio.io/latest/blog/2023/ada-logics-security-assessment/#fuzzing